### PR TITLE
Potholes - Highway

### PR DESCRIPTION
**Pull request by dirkkelly**

### CODE REVIEW

**Comment by dirkkelly** [comment]
https://github.com/BetaNYC/citygram-services-nyc/issues/35

**Comment by dirkkelly** [comment]
screen shot 2015-10-24 at 1 39 52 pm

**Comment by dirkkelly** [comment]
https://citygram-srvcs-nyc-dev-pr-36.herokuapp.com/nyc-311-potholes

File: lib/spy_glass/registry/nyc-311-potholes.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
 require 'spy_glass/registry'
 
+last_week_floating_timestamp = 7.days.ago.utc.iso8601.gsub(/Z$/,'')
```
Comment (j16r) [inline lib/spy_glass/registry/nyc-311-potholes.rb]: Wanna put this in all the registry entries?

**Comment by dirkkelly** [comment]
screen shot 2015-10-24 at 1 49 24 pm

**Pull request merged**